packages/dd-trace/test/appsec/rasp/resources/shi-app/index.js

```diff
@@ -39,6 +39,34 @@ app.get('/shi/execSync/out-of-express-scope', async (req, res) => {
   })
 })
 
+app.get('/cmdi/execFileSync', async (req, res) => {
+  childProcess.execFileSync('sh', ['-c', req.query.dir])
+
+  res.end('OK')
+})
+
+app.get('/cmdi/execFileSync/out-of-express-scope', async (req, res) => {
+  process.nextTick(() => {
+    childProcess.execFileSync('sh', ['-c', req.query.dir])
```

Comment:
I think that `dir` does not make sense in this context, `sh -c` requires a command, not a directory.

Reply:
Done